test/src/test/java/hudson/cli/ConsoleCommandTest.java

```diff
@@ -184,10 +194,15 @@
     }
 
     @Test public void consoleShouldSuccessWithFollow() throws Exception {
-
         FreeStyleProject project = j.createFreeStyleProject("aProject");
-        project.getBuildersList().add(new Shell("echo start - ${BUILD_NUMBER}\nsleep 10s\n"
-                + "echo after sleep - ${BUILD_NUMBER}"));
+        if(Functions.isWindows()) {
+            // the ver >NUL is to reset ERRORLEVEL so we don't fail (ping causes the error)
+            project.getBuildersList().add(new BatchFile("echo start - %BUILD_NUMBER%\r\n"
+                    + "ping 1.1.1.1 -n 1 -w 10000 >NUL\r\nver >NUL\r\necho after sleep - %BUILD_NUMBER%"));
```

Comment:
I switched to timeout, but it doesn't work with jenkins because it tries to redirect stdin and throws an error.